IdSharp.Tagging/ID3v2/Classes/FrameContainer.cs

```diff
@@ -167,9 +167,10 @@ internal void Read(Stream stream, ID3v2TagVersion tagVersion, TagReadingInfo tag
             }
 
             // Process iTunes comments
-            foreach (var comment in new List<IComments>(m_CommentsList))
+            for (int i = m_CommentsList.Count - 1; i >= 0; i--)
             {
-                if (comment.Description != null && comment.Description.StartsWith("iTun"))
+                IComments comment = m_CommentsList[i];
+                if (comment.Description?.StartsWith("iTun", StringComparison.Ordinal) == true)
                 {
                     m_CommentsList.Remove(comment);
                     m_iTunesCommentsList.Add(comment);
```

Comment:
Nice improvement. The only catch is this puts the `m_iTunesCommentsList` in reverse order.